bundles/org.openhab.binding.adorne/src/main/resources/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="adorne"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="hub">
+		<label>Adorne Hub</label>
+		<description>The Adorne Hub serves as the bridge to control Adorne switches, dimmer switches and outlets.</description>
+
+		<config-description>
+			<parameter name="host" type="text" required="false">
+				<label>Host</label>
+				<description>
+					Host name or IP address.
+				</description>
+			</parameter>
+			<parameter name="port" type="integer" required="false">
+				<label>Port</label>
+				<description>
+					Port 
```

Comment:
Remove if you don't provide additional information

Reply:
Done